src/SFA.DAS.EmployerCommitmentsV2.Web/Validators/SelectProviderViewModelValidator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using FluentValidation;
+using SFA.DAS.EmployerCommitmentsV2.Web.Models.CreateCohort;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Validators
+{
+    public class SelectProviderViewModelValidator : AbstractValidator<SelectProviderViewModel>
+    {
+        public SelectProviderViewModelValidator()
+        {
+            RuleFor(x => x.ProviderId)
+                .Must(BeALongGreaterThanZero)
+                .WithMessage("Check UK Provider Reference Number");
+            RuleFor(x => x.EmployerAccountLegalEntityPublicHashedId)
+                .NotNull()
+                .NotEmpty();
```

Comment:
do you need a message for this one?

Reply:
This value is set from the query string and isn't something the user explicitly provides. The search box only accepts a provider id and only shows validation for the provider id based on the screen shot.